### DISCUSSION TITLE
Are boolean variants supported?

### DISCUSSION THREAD

**Discussion by JameelKhan9** [joe-bell/cva#79] [Q&A] [answered]
Are boolean variants currently supported? i.e. disabled, fullWidth, partyMode, etc.

I have a feeling this is a bigger conversation around how non-string based props can influence variants.

**Comment by joe-bell** [comment]
Yes

**Reply by JameelKhan9** [reply]
Is there any examples?

**Comment by knoefel** [comment]
You can find an example [here (Dynamic Text Content)](https://github.com/joe-bell/cva#other-use-cases)

**Reply by JameelKhan9** [reply]
Using string as a boolean isn't the same, but close enough - thanks for the link.

**Comment by JameelKhan9** [comment, marked as answer]
Updated since docs moved - Using string as a boolean isn't the same, but close enough:
https://cva.style/docs/examples/other-use-cases

**Comment by bjornbrasse** [comment]
No, it's not close enough. When evaluation a prop as a boolean, that's not a string, i.e. dayjs(date).isToday return true of false. It would be great when cva could accept this, and return the true or false variant... Please re-open this request. Thnx.

**Reply by joe-bell** [reply]
Feel free to raise a PR

**Comment by joe-bell** [comment]
Closing this as it will be resolved by #176
